File: legacy-algorithm-study/[프로그래머스 Al]/위장.py
def solution(brown, yellow):
    answer = []
    for i in range(1,int(yellow**0.5)+1):
        if not yellow % i:
            j = yellow // i
            
            if 2*(i+j) +4 == brown:
                answer = [j+2,i+2]
                
    return answer

File: legacy-algorithm-study/[프로그래머스 Al]/위장.js
function solution(brown, yellow) {
    var answer = [];
    
    for(let i = 1 ; i<Math.floor(Math.sqrt(yellow)+1); i++){
        if(yellow % i === 0){
            let j = parseInt(yellow/i,10)
            
            if(2*(i+j)+4 === brown ){
                answer = [j+2,i+2];
            }
        }
        
    }
    return answer;
}
